waves-scripts

```diff
@@ -82,9 +82,10 @@ start_bobtimus () {
 
     set +e
     find ./waves/dist ./waves/dist/index.html &> /dev/null
+    [ $? -ne 0 ] && is_built=false
```

Comment:
Nitpick: Double negations are hard to read.

How about `build_failed=true` and then below you say `if build_failed; then`...